Incoporate windows/privacy teams feedback (#2656)
MicrosoftDocs/edge-developer: microsoft-edge/webview2/concepts/data-privacy.md

---
# Data and privacy in WebView2

> WebView2 collects a set of optional and required diagnostic data to keep WebView2 secure and up to date, diagnose issues, and improve WebView2. By agreeing to the WebView2 Runtime Terms and Conditions License, WebView2 developers acknowledge WebView2 will collect the described data in this document.  To view the license, go to [Download the WebView2 Runtime](https://developer.microsoft.com/microsoft-edge/webview2/#download-section), where clicking any of the download buttons, such as **Get the Link**, **Download**, or **x64**, shows the license in a dialog.

Reviewer: ```suggestion
WebView2 collects a set of optional and required diagnostic data to keep WebView2 secure and up to date, diagnose issues, and improve WebView2. By agreeing to the WebView2 Runtime Terms and Conditions License, WebView2 developers acknowledge that WebView2 will collect the data that's described in this article.  To view the license, go to [Download the WebView2 Runtime](https://developer.microsoft.com/microsoft-edge/webview2/#download-section), where clicking any of the download buttons, such as **Get the Link**, **Download**, or **x64**, shows the license in a dialog.
```
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/t/that-vs-which - "Include that even if the sentence is clear without it. It helps to clarify the sentence structure."

* "article" - this repo leans toward "article", sometimes "page" eg a navigation page, rather than "document".

> Additionally, WebView2 follows the standards that are outlined in the [Microsoft Edge Privacy Whitepaper](../../privacy-whitepaper/index.md).  WebView2 has mechanisms to ensure privacy.  WebView2 data collection follows the same strict standards as Microsoft Edge. For more information see [Microsoft Privacy Statement – Microsoft privacy](https://privacy.microsoft.com/en-us/privacystatement).

Reviewer: ```suggestion
Additionally, WebView2 follows the standards that are outlined in the [Microsoft Edge Privacy Whitepaper](../../privacy-whitepaper/index.md).  WebView2 has mechanisms to ensure privacy.  WebView2 data collection follows the same strict standards as Microsoft Edge. For more information, see [Microsoft Privacy Statement – Microsoft privacy](https://privacy.microsoft.com/privacystatement).
```
* removed en-us, will that work ok?

https://privacy.microsoft.com/en-us/privacystatement
https://privacy.microsoft.com/privacystatement

<!-- ====================================================================== -->
## Optional diagnostic data

> WebView2 is a Windows component and thus follows the Windows diagnostic data collection practices. For more information see [Windows diagnostic data](https://learn.microsoft.com/en-us/windows/privacy/configure-windows-diagnostic-data-in-your-organization).  The data collection consent for WebView2 is governed by the **Settings > Privacy & security > Diagnostics & feedback > Diagnostic data** setting on Windows 10 and Windows 11 as a centralized switch.

Reviewer: ```suggestion
WebView2 is a Windows component and thus follows the Windows diagnostic data collection practices. For more information, see [Windows diagnostic data](/windows/privacy/configure-windows-diagnostic-data-in-your-organization).  The data collection consent for WebView2 is governed by the **Settings > Privacy & security > Diagnostics & feedback > Diagnostic data** setting on Windows 10 and Windows 11 as a centralized switch.
```

https://learn.microsoft.com/en-us/windows/privacy/configure-windows-diagnostic-data-in-your-organization
